api/resources/serializers.py

```diff
@@ -72,6 +78,68 @@ def create(self, validated_data):
         try:
             root_outcome = Outcome.objects.for_registration(primary_registration, create=True)
         except NoPIDError:
-            raise Conflict('Cannot add Resources to a Registration that does not have a DOI')
+            raise Conflict('Cannot add Resrouces to a Registration without a DOI')
 
         return OutcomeArtifact.objects.create(outcome=root_outcome)
+
+    def update(self, instance, validated_data):
+        updated_title = validated_data.get('title')
+        if updated_title is not None:
+            instance.title = updated_title
```

Comment:
Minor/Clarity: didn't we decide that `title` might be auto-populated but not user-inputted?